test/routes/token.js

```diff
@@ -23,7 +23,7 @@ function joiNotAllowed(err, param) {
   assert.equal(err.details[0].message, `"${param}" is not allowed`);
 }
 
-describe('/token POST', function () {
+describe.only('/token POST', function () {
```

Comment:
Actually, what's this `only`, is it a testing leftover?

Reply:
> is it a testing leftover? 

ah yes removing..